<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AAL projekt - Piotr Gwizdała</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Ścieżka do pliku wejściowego</string>
    </property>
    <widget class="QPushButton" name="startPath">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pathInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>in1.txt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wpisz poniżej ścieżkę do pliku</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Przeglądaj...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Badanie złożoności obliczeniowej</string>
    </property>
    <widget class="QLineEdit" name="alfabetSizeInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Długość alfabetu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dataBeginInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Początkowy rozmiar problemu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dataEndInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Końcowy rozmiar problemu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dataStepInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kwant wzrostu problemu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startComp">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="kInput1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parametr k</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="repsInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ilość powtórzeń dla każdego rozmiaru</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>271</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>Tekst własny z podglądem drzewa</string>
    </property>
    <widget class="QLineEdit" name="userTextInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>barbara</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wpisz poniżej tekst własny</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parametr k</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="kInput2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startOwn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>251</width>
       <height>411</height>
      </rect>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Dane wygenerowane</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rozmiar problemu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="alfabetSize4Gen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parametr k</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startGen">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Długość alfabetu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="wordLength4Gen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>32</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="k4Gen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="tb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>571</width>
      <height>231</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Tutaj będą wyświetlone rezultaty działania programu.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>561</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Program ten umożliwia przetestowanie algorytmu znajdywania najdłuższego tekstu powtarzającego się co najmniej k razy w tekście wejściowym. Testowany algorytm opiera się na drzewie sufiksowym konstruowanym algorytmem Ukkonen'a.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
